src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -196,16 +196,28 @@ private void appendOneNativeArgument(ExecutionContext context, object obj, char
                         if (NeedQuotes(arg))
                         {
```

Comment:
How about:
```csharp
_arguments.Append('"'); 
_arguments.Append(arg); 
for (int i = arg.Length-1; i >= 0 && arg[i] == '\\'; i--)
{
    _arguments.Append('\\');
}
_arguments.Append('"'); 
```
?

Reply:
That's much better than my attempt.  Will change.